src/main/connector/system/Windows/Windows.yaml

```diff
@@ -133,26 +143,68 @@ monitors:
           system.filesystem.usage{system.filesystem.state=used}: $3
   service:
     simple:
-      type: multiInstance
       sources:
-        # Name,ProcessId,StartMode,State
-        source(1):
+        # Name;ProcessId;StartMode;State
+        serviceInformation:
           type: wmi
           namespace: root\CIMv2
-          query: SELECT Name,ProcessId,StartMode,State From Win32_Service
+          query: SELECT Name,ProcessId,State FROM Win32_Service WHERE StartMode != 'Disabled'
+      mapping:
+        source: ${source::monitors.service.simple.sources.serviceInformation}
+        attributes:
+          id: $1
+          processId: $2
+        metrics:
+          system.service.state: $3
+  pageFile:
+    simple:
+      sources:
+        # Name;CurrentUsage;CurrentUsage
+        pageFileUsageInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT Name,CurrentUsage,CurrentUsage FROM Win32_PageFileUsage
+        # Name;MaximumSize
+        pageFileSettingsInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: SELECT Name,MaximumSize FROM Win32_PageFileSetting
+        #Name;CurrentUsage;Utilization;Name;MaximumSize
+        pageFileCombinedInformation:
+          type: tableJoin
+          leftTable: ${source::monitors.pageFile.simple.sources.pageFileUsageInformation}
+          rightTable: ${source::monitors.pageFile.simple.sources.pageFileSettingsInformation}
+          leftKeyColumn: 1
+          rightKeyColumn: 1
           computes:
-          - type: translate
+          - type: divide
             column: 3
-            translationTable: ${translation::serviceStatusTranslationTable}
+            value: $5
       mapping:
-        source: ${source::monitors.service.simple.sources.source(1)}
+        source: ${source::monitors.pageFile.simple.sources.pageFileCombinedInformation}
+        attributes:
+          id: $1
+        metrics:
+          system.paging.usage: $2
+          system.paging.utilization: $3
+  network:
+    simple:
+      sources:
+        # Name;PacketsOutboundDiscarded;PacketsReceivedDiscarded;PacketsSentPersec;PacketsReceivedPersec;PacketsOutboundErrors;PacketsReceivedErrors
+        networkInformation:
+          type: wmi
+          namespace: root\CIMv2
+          query: >
+            SELECT Name,PacketsOutboundDiscarded,PacketsReceivedDiscarded,PacketsSentPersec,PacketsReceivedPersec,PacketsOutboundErrors,PacketsReceivedErrors

```

Comment:
Avoid using `PerfFormattedData`. We should use `PerfRawData` WMI classes instead, to get actual counters instead of their rates (we don't know for which timestamps they've been calculated for).